src/core/geometry/path/fromPointArray.test.js

```diff
@@ -0,0 +1,23 @@
+const fromPointArray = require('./fromPointArray')
+const toPointArray = require('./toPointArray')
+const test = require('ava')
+const vec3 = require('../../math/vec3')
+
+test('fromPointArray: Creating a path from no points produces an empty, non-canonical path', t => {
+  const created = fromPointArray({}, [])
+  t.false(created.isCanonicalized)
+  t.deepEqual(toPointArray({}, created), [])
+})
+
+test('fromPointArray: Creating a path from one point produces a non-canonical path with that element', t => {
+  const created = fromPointArray({}, [[1, 1, 0]])
+  t.false(created.isCanonicalized)
+  t.deepEqual(toPointArray({}, created), [vec3.fromValues(1, 1, 0)])
+})
+
+test('fromPointArray: Creating a closed path from one point produces a closed non-canonical path with that element', t => {
```

Comment:
a path with one point is not a path but a point technicaly, minimal length requirement when creating from points should be 2

Reply:
I think that we need support across the board for empty geometries.

Also path with one point may be isomorphic to a point, but that doesn't mean that it isn't also a path, and capable of being composed into longer paths.

Given this, I'd say that the minimum length for a path is zero.

Reply:
what are use cases for empty geometries ?

Reply:
The same use cases for empty lists, and so on.

They allow a unified interface that supports composition and decomposition without exceptional cases.

Consider a function which concatenates a leaf shape to a path.
If you can't pass it an empty path, it can't concatenate it.

Consider a function which conditionally generates a leaf to concatenate.
If it can't return an empty path, you need an additional channel to communicate that decision, and additional logic to avoid the composition.

With empty paths both cases are handled properly with unconditional concatenation.

addLeaf(path.create()) and path.concat(buildLeaf()) respectively.